app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    com_ls = command.split()
```

Comment:
The checklist specifies that you should check if the input string contains exactly three elements separated by space and starts with 'mv'. Consider adding a validation step here to ensure these conditions are met.